fe/src/components/common/TextArea/TextAreaContainer.tsx

```diff
@@ -0,0 +1,37 @@
+import styled from "styled-components";
+import FileUploadArea from "./FileUploadArea";
+import TextAreaWrapper from "./TextAreaWrapper";
+
+export default function TextAreaContainer({
+  value,
+  appendContent,
+  ...props
+}: {
+  value: string;
+  appendContent: (content: string) => void;
+} & React.TextareaHTMLAttributes<HTMLTextAreaElement>) {
+  return (
+    <StyledTextAreaContainer>
+      <TextAreaWrapper {...{ value, ...props }} />
+      <FileUploadArea {...{ appendContent }} />
```

Comment:
객체 한 겹 벗겨도 될 것 같은데 요렇게 해도 잘 동작하나요?

```suggestion
      <TextAreaWrapper { value, ...props } />
      <FileUploadArea { appendContent } />
```

Reply:
엇 아니요..!
아래 방식 중 하나로 작성해줘야 하네요.
```
<FileUploadArea appendContent={appendContent} />
<FileUploadArea {...{ appendContent }} />
```

Reply:
아앙 appendContent가 객체인줄 알았네요 굳굳!